modules/assim.batch/R/pda.utils.R

```diff
@@ -256,12 +256,23 @@ pda.load.priors <- function(settings, con, path.flag = TRUE) {
     for (i in seq_along(settings$pfts)) {
       
       files <- dbfile.check("Posterior", settings$pfts[[i]]$posteriorid, con, settings$host$name)
```

Comment:
Oh wait, in addition to the problem I mention below, I think there is another bug here

this line should be 
```
files <- dbfile.check("Posterior", settings$assim.batch$prior$prior.id[[i]] con, settings$host$name)
```